source/system/Entry.js

```diff
@@ -376,6 +391,11 @@ Entry.createNew = function(archive, groupID) {
     );
     // get the raw dataset for the new entry
     var entry = searching.findEntryByID(westley.getDataset().groups, id);
+
+    // init default attributes
+    entry.attributes = entry.attributes || {};
+    Object.assign(entry.attributes, entryTools.setDefaultAttributesValues());
```

Comment:
Not sure why we'd need any defaults. That creates quite a bit of overhead, and it's important to remember that no one currently has any entries with password expiration attributes in the wild. We should primarily support the absence of password expiration attributes, and then detect if they're correctly set.. nothing in between.